src/components/Person/Person.jsx

```diff
@@ -1 +1,31 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person: {
+  name,
+  age,
+  isMarried,
+  partnerName,
+  sex,
+} }) => {
+  const partnerStatusName = sex === 'f'
+    ? 'husband'
+    : 'wife';
```

Comment:
Next time, avoid using "magic values":
```js
const FEMALE = 'f';
const HUSBAND = 'husband';
const WIFE = 'wife';
...
const partnerStatusName = sex === FEMALE ? HUSBAND : WIFE;
```
This will increase the readability of the code and protect further work from errors.